src/App.tsx

```diff
@@ -14,7 +24,7 @@ export const App = () => {
       </div>
 
       <div className="sidebar">
-        <FindMovie />
+        <FindMovie onClick={addMovie} />
```

Comment:
I didn't really understand the comment, because that's exactly what I'm doing: passing a property that allows me to add a movie to the movie list in the App component. Shoud i just rename "onClick"?